src/deployment/azuredeploy.json

```diff
@@ -456,16 +410,16 @@
                 {
                     "apiVersion": "2015-11-01-preview",
                     "location": "[resourceGroup().location]",
-                    "name": "[concat('VMInsights', '(', variables('monitorAccountName'), ')')]",
+                    "name": "[concat('VMInsights', '(', parameters('name'), ')')]",
```

Comment:
This concatenated `VMInsights(<instance-name>)` value seems to get recreated in a few places. Can we define it once, in a template `variable()`, and just reuse that?

Reply:
It may also be worth keeping `monitorAccountName` as a variable, but just making it exactly equal to `parameters('name')`. That way it is easy to change in the future, see where it gets used, &c.

Reply:
These are both good ideas. I've updated the PR to reflect the suggestion.